packages/gestalt/src/Table.css

```diff
@@ -7,40 +7,54 @@
 
 .th {
   composes: alignStart from "./Typography.css";
-  composes: paddingY3 from "./boxWhitespace.css";
-  composes: paddingX3 from "./boxWhitespace.css";
 }
 
+.th,
 .td {
   composes: paddingY3 from "./boxWhitespace.css";
   composes: paddingX3 from "./boxWhitespace.css";
 }
 
+.stickyHeader {
+  box-shadow: var(--elevation-raised-top);
+}
+
+.stickyFooter {
+  box-shadow: var(--elevation-raised-bottom);
+}
+
+.stickyHeader,
 .stickyHeader tr th {
-  background-color: var(--g-colorGray0);
-  position: sticky;
   top: 0;
-  z-index: 2;
 }
 
+.stickyFooter,
 .stickyFooter tr td {
-  background-color: var(--g-colorGray0);
   bottom: 0;
-  position: sticky;
-  z-index: 2;
 }
 
 .columnSticky {
-  background-color: var(--g-colorGray0);
   left: 0;
-  position: sticky;
   z-index: 1;
 }
 
-.stickyHeader .columnSticky {
-  z-index: 3;
+.stickyHeader,
+.stickyFooter,
+.stickyHeader tr th,
+.stickyFooter tr td {
+  z-index: 2;
+}
+
+.columnSticky,
+.stickyHeader,
+.stickyFooter,
+.stickyHeader tr th,
+.stickyFooter tr td {
+  background-color: var(--g-colorGray0);
```

Comment:
Similar note here and throughout: where we're already declaring these styles in generic classes, let's compose those to create the styles needed here

Reply:
DONE

Reply:
composition only allowed with single